rllib/agents/a3c/a2c.py

```diff
@@ -40,19 +42,105 @@
 
 logger = logging.getLogger(__name__)
 
-A2C_DEFAULT_CONFIG = merge_dicts(
-    A3C_CONFIG,
-    {
-        "rollout_fragment_length": 20,
-        "min_time_s_per_reporting": 10,
-        "sample_async": False,
-        # A2C supports microbatching, in which we accumulate gradients over
-        # batch of this size until the train batch size is reached. This allows
-        # training with batch sizes much larger than can fit in GPU memory.
-        # To enable, set this to a value less than the train batch size.
-        "microbatch_size": None,
-    },
-)
+
+class A2CConfig(A3CConfig):
+    """Defines a A2CTrainer configuration class from which a new Trainer can be built.
+
+    Example:
+        >>> config = A2CConfig().training(lr=0.01, grad_clip=30.0)\
```

Comment:
I think we should make it a habit (well, we need to make doctests happy, too) to have imports in all docstrings. Here the one of the config itself is missing, below we need that of the config and tune.